Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[workspace]
+
+members = [
+    "base_layer/blockchain",
```

Comment:
For some reason, most chains I've seen call this folder `chain`

Reply:
Personally, I prefer the more explicit `blockchain`, but not married to it